client/a8c-for-agencies/sections/referrals/primary/referrals-overview/style.scss

```diff
@@ -7,6 +7,12 @@ $data-view-border-color: #f1f1f1;
 	margin-block-end: 48px;
 }
 
+.referrals-overview__notice-button {
+	margin-block-start: 1rem;
+	background-color: var(--color-accent-100) !important;
+	color: var(--color-surface) !important;
```

Comment:
It is not recommended to use `!important`. I think we can do `.button..referrals-overview__notice-button` to take precedence.

Reply:
Good point. Thank you!